packages/cardpay-reward-programs/cardpay_reward_programs/rules/retro_airdrop.py

```diff
@@ -0,0 +1,92 @@
+import pandas as pd
+from cardpay_reward_programs.config import default_payment_list
+from cardpay_reward_programs.rule import Rule
+
+
+class RetroAirdrop(Rule):
+    """
+    reward_per_transaction = (total_n_payments in start_block< x <= end_block/ total_n_payments from  start_snapshot_block < x <= end_snapshot_block)
+    reward_per_payee (amount) = (total_n_payments of payee in start_block < x <= end_block) * reward_per_transaction
```

Comment:
I don't think this is correct - the start & end block are irrelevant here, it's the snapshot ones we care about for the calculation.

Start & end blocks we talked about being the range in which the *rule calculation can be triggered* but here we're doing just one - the start / end for the *snapshot* blocks are about what data range we care about in the calculation.